### PR TITLE
add CLI for getting TLF crypt keys from kbfs

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

**Comment by songgao** [comment]
Related: https://github.com/keybase/kbfs/pull/226

File: go/client/cmd_tlf.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	}
+}
+
+func (c *cmdTlfCryptKeys) Run() error {
+	tlfClient, err := GetTlfClient(c.G())
+	if err != nil {
+		return err
+	}
+
+	if err = RegisterProtocols(nil); err != nil {
```
Comment (maxtaco) [inline go/client/cmd_tlf.go]: call `RegisterProtocolWithGlobalContext` instaed.

File: go/service/tlfKeys.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	rpc "github.com/keybase/go-framed-msgpack-rpc"
+)
+
+type tlfHandler struct {
+	*BaseHandler
+	libkb.Contextified
+}
+
+func newTlfHandler(xp rpc.Transporter, g *libkb.GlobalContext) *tlfHandler {
+	fmt.Println("haha")
```
Comment (maxtaco) [inline go/service/tlfKeys.go]: whoops!

File: go/client/cmd_tlf.go (hunk 1 above)
Comment (songgao) [inline go/client/cmd_tlf.go]: It seems it's actually what `RegisterProtocols` is doing: https://github.com/keybase/client/blob/e84f240a9e89fce5b264399b37f6f85c78c09e30/go/client/rpc.go#L108

Or did you mean using `RegisterProtocolsWithContext`?

Comment (maxtaco) [inline go/client/cmd_tlf.go]: yes

**Comment by maxtaco** [comment]
👍

**Comment by songgao** [comment]
Change the protocol interface a little bit per https://github.com/keybase/kbfs/pull/226#discussion_r74166512

@maxtaco could you take another look? Thanks!

**Comment by maxtaco** [comment]
👍

**Comment by patrickxb** [comment]
we're showing this command to the public?

**Comment by patrickxb** [comment]
sorry, didn't see it was in commands_devel...ignore.

**Pull request merged**